### PR TITLE
yocto/docker/Dockerfile: Enable HSM token support

### PR DESCRIPTION
**Pull request by davidsonntag**
Update the Docker Image to Debian Bullseye.
Enable support for HSM PKCS#11 tokens.

### CODE REVIEW

File: yocto/docker/run-docker.sh [hunk 1]
```diff
@@ -13,11 +13,20 @@ if [ "$2" ];then
 	EXTRA_ARGS="--volume $2:/tmp/sshagent --env=SSH_AUTH_SOCK=/tmp/sshagent"
 fi
 
+if [ ! "$3" ]; then
```
Comment (quitschbo) [inline yocto/docker/run-docker.sh]: make this optional, not every one uses this same for pki

File: yocto/docker/run-docker.sh [hunk 2]
```diff
@@ -13,11 +13,20 @@ if [ "$2" ];then
...
+	echo "Wrong usage: provide mirror build directory as parameter"
+	exit
+fi
+
+
 docker run \
  -it \
 ${EXTRA_ARGS} \
+--mount type=bind,source=/var/run/pcscd,target=/var/run/pcscd \
```
Comment (quitschbo) [inline yocto/docker/run-docker.sh]: this should also be an optional argument

**Review by quitschbo — CHANGES_REQUESTED** [review]

File: yocto/docker/run-docker.sh (hunk 1 above)
Comment (davidsonntag) [reply yocto/docker/run-docker.sh]: is optional now.

File: yocto/docker/run-docker.sh (hunk 2 above)
Comment (davidsonntag) [reply yocto/docker/run-docker.sh]: is optional now

**Review by quitschbo — APPROVED** [review]
Looks good. Thanks for your contribution.

**Pull request merged**